KeyVaultRecoverySamples.cs

```diff
@@ -70,7 +69,7 @@ public static async Task DemonstrateRecoveryAndPurgeForNewVaultAsync()
                 Console.WriteLine("done.");
 
                 Console.Write("Retrieving newly created vault...");
-                var retrievedVault = await sample.ManagementClient.Vaults.GetAsync(rgName, vaultName).ConfigureAwait(false);
+                var retrievedVault = (await sample.ManagementClient.Vaults.GetAsync(rgName, vaultName).ConfigureAwait(false)).Value;
```

Comment:
Like the sample above, I'd stick with `Response<T>` here and use its `Value` property only when needed.